.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-yarn lint-staged
+yarn install && yarn lint-staged
```

Comment:
I don't think yarn install is desirable here - it can potentially create unexpected changed and affect yarn.lock, which would necessitate another commit cycle.

The lint-staged came from renovate-bot, perhaps we can ignore it and update this dep manually?

Reply:
Nope, `lint-staged` is used by us.
We can simply to that with a normal script. let me remove this and make another PR about this.

Reply:
Yes, just need to run `yarn install` after every merge from main